src/atomate2/forcefields/jobs.py

```diff
@@ -309,6 +309,94 @@ class GAPRelaxMaker(ForceFieldRelaxMaker):
         Keyword arguments that will get passed to :obj:`Relaxer()`.
     task_document_kwargs : dict
         Additional keyword args passed to :obj:`.ForceFieldTaskDocument()`.
+    potential_param_filename: str | Path
+        param_file_name for :obj: mace.calculators.MACECalculator()'.
+    potential_device: str
+        device for :obj: mace.calculators.MACECalculator()'.
+    potential_kwargs: dict
+        Further kwargs for :obj: mace.calculators.MACECalculator()'.
+    """
+
+    name: str = "MACE relax"
+    force_field_name: str = "MACE"
+    relax_cell: bool = False
+    steps: int = 500
+    relax_kwargs: dict = field(default_factory=dict)
+    task_document_kwargs: dict = field(default_factory=dict)
+    potential_param_file_name: str = "MACE.model"
+    potential_device: str = "cpu"
+    potential_kwargs: dict = field(default_factory=dict)
+
+    def _relax(self, structure: Structure) -> dict:
+        from mace.calculators import MACECalculator
+
```

Comment:
Try 32 or 64 might fix the failing test.

```suggestion
self.potential_kwargs.setdefault("default_dtype", "float32")
```

Reply:
Depending on what precision `MACE.model` was trained with. [Helped in MBD](https://github.com/janosh/matbench-discovery/commit/467d77746c1f7a445b035a02db97ccc832af7dd1#diff-7c0d3ddd75d72c03759aba8249fe98bd31f16bfafe5945b11c7fa3754deabe61L99).